live-build/misc/live-build-hooks/90-raw-disk-image.binary

```diff
@@ -120,6 +120,21 @@ rsync --info=stats3 -Wa binary/* /mnt/
 zfs create -V 4G -b "$(getconf PAGESIZE)" \
 	-o logbias=throughput -o sync=always \
 	-o primarycache=metadata rpool/swap
+
+#
+# We need to wait for the /dev/zvol device to appear before we proceed; this
+# can take a few seconds, so we retry until it succeeds.
+#
+tries=0
+while [[ ! -e /dev/zvol/rpool/swap && $tries -lt 5 ]]; do
```

Comment:
Are we sure that waiting a maximum of 25 seconds is enough time to guarantee the device will appear when everything is working correctly?

I'd rather wait a long time and have it eventually work, than have to deal with flaky failures due to not waiting long enough. If 25 seconds is sufficient, I'm good with this; if we're not sure, I'd suggest bumping this to something on the order of minutes (I assume minutes is long enough, since we're starting with seconds here).

Reply:
When I was looking at the same issue in the app-gate, I think I found that the device typically appeared within a couple ms, and the outliers were still less <100ms. I think I ended up setting the timeout there to 20s.

Reply:
Perfect, thanks! Based on that, I'm good with this.